homeassistant/components/lcn/websocket.py

```diff
@@ -158,6 +158,11 @@ async def websocket_get_entity_configs(
     else:
         entity_configs = config_entry.data[CONF_ENTITIES]
 
+    for entity_config in entity_configs:
+        if (entity := get_entity_entry(hass, entity_config, config_entry)) is None:
+            continue
+        entity_config[CONF_NAME] = entity.name or entity.original_name
+
```

Comment:
This will mutate config_entry.data since you have a reference to an element in the CONF_ENTITIES. From line 159.

Reply:
Thanks for pointing that out to me. I fixed it.